test/org/sagebionetworks/bridge/play/controllers/UserDataDownloadControllerTest.java

```diff
@@ -54,7 +51,7 @@ public void test() throws Exception {
 
         // validate args sent to mock service
         ArgumentCaptor<DateRange> dateRangeCaptor = ArgumentCaptor.forClass(DateRange.class);
-        verify(mockService).requestUserData(eq(studyIdentifier), same(user), dateRangeCaptor.capture());
+        verify(mockService).requestUserData(eq(studyIdentifier), any(String.class), dateRangeCaptor.capture());
```

Comment:
Since we changed from passing in the user to passing in the email from the participant, we should update the test to use Participant and verify the email from Participant is being passed into requestUserData().